packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -38,6 +38,7 @@ type FlagsT = {
   port: number;
   terminal: string | undefined;
   xcconfig?: string;
+  iosBuildFolder?: string;
```

Comment:
it's already a "run ios" command, `buildFolder` would be descriptive enough :)

Reply:
Makes sense 👍